src/main/java/liquibase/ext/cassandra/database/CassandraDatabaseConnection.java

```diff
@@ -0,0 +1,65 @@
+package liquibase.ext.cassandra.database;
+
+import com.ing.data.cassandra.jdbc.CassandraDriver;
+import liquibase.Scope;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.DatabaseException;
+
+import java.net.URI;
+import java.sql.Driver;
+import java.util.Properties;
+
+public class CassandraDatabaseConnection extends JdbcConnection {
+
+    @Override
+    public int getPriority() {
+        return 201;
```

Comment:
Usually we do CONSTANT + increment, but we can fix it later.

Reply:
Thanks for the recommandation @filipelautert: I didn't know how was fixed this value, so I took a value for which I was almost sure the `CassandraDatabaseConnection` will be used. I can fix that in a separate PR, as you suggested; do you have an example of the good way to do that?

Reply:
@maximevw at liquibase-core we have PRIORITY_DATABASE + 5, like in https://github.com/liquibase/liquibase/blob/53db8b6d12a9db371bbcd620f99aca954d3bbbfd/liquibase-standard/src/main/java/liquibase/database/core/EnterpriseDBDatabase.java#L46 . But could be +200, no problems!